src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "ms-blazorwasm-companion",
+  "displayName": "Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension",
+  "description": "Companion extension for debugging Blazor WebAssembly",
+  "version": "0.0.1",
+  "engines": {
+    "vscode": "^1.52.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "extensionKind": [
+    "ui"
+  ],
+  "publisher": "ms-dotnettools",
+  "activationEvents": [
+    "onCommand:ms-blazorwasm-companion.launchDebugProxy"
+  ],
+  "main": "./dist/extension.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "ms-blazorwasm-companion.launchDebugProxy",
+        "title": "Launch Blazor WebAssembly Debugging Proxy"
```

Comment:
I forget how this functions, can you ensure this does not show up in the ctrl + shift + p dialog?